src/util/QdmAttributeHelpers.ts

```diff
@@ -157,10 +157,11 @@ export const stringifyValue = (value, topLevel = false, codeSystemMap = {}) => {
   if (!value) {
     return "null";
   }
-
   if (value instanceof cqmModels.CQL.Code) {
     const title = codeSystemMap[value.system] || value.system;
     return `${title} : ${value.code}`;
+  } else if (value.unit == "%") {
+    return `${value.value} ${value.unit}`;
```

Comment:
I am unable to reproduce this issue initially ?
If we have to check for "%" then what about all other units ?

Reply:
<img width="404" alt="image" src="https://github.com/MeasureAuthoringTool/madie-patient/assets/47760891/c7ccdf42-80fb-4437-8ace-b32e5edd0aa5">
This is what it looks like right now, adding the specific check for % prevents it